### ISSUE TITLE
After updating SDK version its showing loader only 

### ISSUE THREAD

**Issue by parulgarg004** [payfort/fort-ios-sdk#13]
![Simulator Screen Shot - iPhone 13 Pro Max - 2022-05-31 at 16 05 40](https://user-images.githubusercontent.com/5426160/171156590-02b7e859-aaae-440d-9182-5ad2a98a2d22.png)

I have updated the SDK version and using objective-c. After reading the documentation everything is updated and code is compiling. But the issue is after getting token it keep on showing the initialisation part. Code part is also attached. 
I'm attaching code snippet below:

`  PayFortController *payFort = [[PayFortController alloc] initWithEnviroment:PayFortEnviromentSandBox];
    payFort.presentAsDefault = UIModalPresentationFullScreen;
	payFort.isShowResponsePage = YES;
	[payFort setPayFortCustomViewNib:@"PayFortView"];
    NSMutableDictionary *request = [[NSMutableDictionary alloc] init];
    [request setValue:[NSString stringWithFormat:@"%.0f",_orderPrice*100] forKey:kPayFortAmount];
    [request setValue:vPayFortComandPurchace forKey:kPayFortComand];
    [request setValue:vPayFortCurrencySAR forKey:kPayFortCurency];
    [request setValue:CurrentUser.email forKey:kPayFortCustomerEmail];
    if (layoutRTL) {
        [request setValue:vPayFortLanguageArabic forKey:kPayFortLanguage];
    }
    else {
        [request setValue:vPayFortLanguageEnglish forKey:kPayFortLanguage];
    }
    [request setValue:_orderId forKey:kPayFortOrderIdentifier];
    [request setValue:sdkToken forKey:kPayFortSDKToken];
   	
    [payFort callPayFortWithRequest:request currentViewController:appDelegate.window.rootViewController success:^(NSDictionary *requestDic, NSDictionary *responeDic) {
        NSError *err1 = nil;
        SARPayFort *paymentResponse = [[SARPayFort alloc] initWithDictionary:responeDic error:&err1];
        if ([paymentResponse.status isEqualToString:@"14"]) {
            appDelegate.callbackBlock(paymentResponse,nil);
        }
        else {
            if(paymentResponse == nil)
                appDelegate.callbackBlock(nil,[NSError errorWithDomain:@"Payment Failed" code:99 userInfo:nil]);
            else
                appDelegate.callbackBlock(nil,[NSError errorWithDomain:paymentResponse.response_message code:99 userInfo:nil]);
        }
    } canceled:^(NSDictionary *requestDic, NSDictionary *responeDic) {
        NSError *err1 = nil;
        SARPayFort *paymentResponse = [[SARPayFort alloc] initWithDictionary:responeDic error:&err1];
        if(paymentResponse == nil)
            appDelegate.callbackBlock(nil,[NSError errorWithDomain:@"Payment Cancelled" code:99 userInfo:nil]);
        else
            appDelegate.callbackBlock(nil,[NSError errorWithDomain:paymentResponse.response_message code:99 userInfo:nil]);
    } faild:^(NSDictionary *requestDic, NSDictionary *responeDic, NSString *message) {
        NSError *err1 = nil;
        SARPayFort *paymentResponse = [[SARPayFort alloc] initWithDictionary:responeDic error:&err1];
        if(paymentResponse == nil)
            appDelegate.callbackBlock(nil,[NSError errorWithDomain:@"Payment Failed" code:99 userInfo:nil]);
        else
            appDelegate.callbackBlock(nil,[NSError errorWithDomain:paymentResponse.response_message code:99 userInfo:nil]);
    }];`

@mohannadsaleh @cetozgen can you please check, thank you.

**Comment by parulgarg004** [comment]
@mohannadsaleh @cetozgen can you please check, thank you.

**Comment by cetozgen** [comment]
Hi @parulgarg004 

Could you please declare PayFortController *payFort at class level and initialize the object only one time on method such as onload ?

Regards,
Ozgen

**Comment by parulgarg004** [comment]
@cetozgen I tried it and seems to load the view now, but there is another crash. We have custom view created which was earlier linked with PayFortView.h and now this class is not available in the new package. Can you provide me solution for this? 

<img width="1420" alt="Screenshot 2022-06-02 at 5 53 33 PM" src="https://user-images.githubusercontent.com/5426160/171628424-8987907d-27b2-4f50-88b9-806195388f4a.png">

**Comment by hussainashraf4u** [comment]
Same issue, any update on this?

**Comment by parulgarg004** [comment]
@cetozgen I'm working on it again and now the app started crashing after showing an initialization pop-up, can you please help at the earliest? 
![Simulator Screen Shot - iPhone 14 Pro - 2022-09-22 at 19 22 25](https://user-images.githubusercontent.com/5426160/191765763-63be7435-123e-4cbe-82cf-7b86126f4ae2.png)

**Comment by cetozgen** [comment]
@parulgarg004 Are you using custom view approach with inheriting .xib from PayfortSDK? There is a limitation forced by the Apple, newer versions of Xcode does not allow you to associate CustomPayfortView with your view.
In order to overcome this, there is a 2nd approach where each UI element is exposed. You can find it on page 30 (https://m.media-amazon.com/images/G/01/APS/api-reference/APS_Integration_Guide_-_iOS.pdf)

**Comment by parulgarg004** [comment]
@cetozgen thanks for quick reply, I will try this and let you know

**Comment by parulgarg004** [comment]
@cetozgen is the sample code updated?

**Comment by cetozgen** [comment]
@parulgarg004 Yes sample code is updated for the latest SDK. If you need further support, kindly create a ticket by sending an email to integration-ps@amazon.com .Our team will be happy to assist you.

**Comment by parulgarg004** [comment]
@cetozgen in sample for custom UI it's showing the same steps as I did. Approach mentioned on page 30 is not updated in the sample.
